webapp-django/crashstats/crashstats/views.py

```diff
@@ -1763,6 +1764,44 @@ def crontabber_state(request, default_context=None):
     return render(request, 'crashstats/crontabber_state.html', context)
 
 
+@utils.json_view
+def crontabber_status(request):
+    api = CrontabberState()
+
+    # start by assuming the status is OK which means no jobs are broken
+    context = {'status': 'OK'}
+
+    all = api.get()['state']
```

Comment:
``all`` is a function in the standard library of Python, I would recommend not using it. (Plus it's a terrible variable name really. :) )

https://docs.python.org/2/library/functions.html#all